deployment/arm/deploy-region.sh

```diff
@@ -44,4 +47,12 @@ az group deployment create \
   --name "Urlist-$location-$(timestamp)" \
   --resource-group $resourceGroupName \
   --template-file region.json \
-  --parameters location=$location apimName=$apimName appServicePlanName=$appServicePlanName frontendAppName=$frontendAppName backendAppName=$backendAppName appInsightsName=$appInsightsName
\ No newline at end of file
+
+  --parameters location=$location apimName=$apimName appServicePlanName=$appServicePlanName frontendAppName=$frontendAppName backendAppName=$backendAppName appInsightsName=$appInsightsName storageActName=$storageActName
+
+echo "Configuring blob storage for static website hosting"
+az storage blob service-properties update \
```

Comment:
Actually - this may get around needing to include the blob stroage / container in the ARM template.  Let's discuss pros/cons tomorrow.